### ISSUE TITLE
Error when using Date::use(CarbonImmutable::class)

### ISSUE THREAD

**Issue by kbond** [laravel/telescope#567]
When using `Date::use(CarbonImmutable::class)` I get the following error when trying to load telescope in the browser:

```
Argument 7 passed to Laravel\Telescope\EntryResult::__construct() must be an instance of Carbon\Carbon, instance of Carbon\CarbonImmutable given, called in /vendor/laravel/telescope/src/Storage/DatabaseEntriesRepository.php on line 93
```

**Comment by virp** [comment]
The same problem, I think you need to change Carbon to CarbonInterface type hint in Telescope package

**Comment by themsaid** [comment]
This was fixed on the 2.0 branch.

If you're using Laravel 5.8 please use Telescope 2.0
